charts/portainer/values.yaml.gotmpl

```diff
@@ -0,0 +1,68 @@
+# Default values for adminer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: portainer/portainer-ce
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: portainer-sa-clusteradmin
+
+podAnnotations: {}
+podLabels: {}
+
+podSecurityContext:
+  {}
+
+securityContext:
+  {}
+
+service:
+  type: "ClusterIP"
+  port: 9000
+
+ingress:
+  enabled: true
+  className: ""
+  annotations:
+    namespace: {{ .Release.Namespace }}
+    cert-manager.io/cluster-issuer: "cert-issuer"
+    traefik.ingress.kubernetes.io/router.entrypoints: websecure
+    traefik.ingress.kubernetes.io/router.middlewares: traefik-traefik-basic-auth@kubernetescrd,traefik-portainer-strip-prefix@kubernetescrd  # namespace + middleware name
```

Comment:
traefik namespace is hard coded. is there a way to use ENV / VAR?

Reply:
I think for traefik it is fine that it is hardcoded. Afterall, we specifically want to fetch these middlewares in other namespaces (example: portainer, adminer) from preciceily the traefik namespace. An alias or env would only be helpful if we switch to a different reverse proxy, which I dont forsee...

Thinking about it again, would you find it acceptable now? :--) Which benefit do you see here with an env-var-alias?